### ISSUE TITLE
Apps for business only supports current channel

### ISSUE THREAD

**Issue by wardkeller** [MicrosoftDocs/OfficeDocs-DeployOffice#1090]
During my testing, I have been unable to get Office 365 Apps for Business to reliably remain on the semi-annual enterprise update channel. I have tried configuring this via group policy, via the commands: "OfficeC2RClient.exe /changesetting channel=semiannual" and "OfficeC2RClient.exe /update user" as well as by using  the office deployment tool. The only method that gets Apps for Business to set its channel to semi-annual is the office deployment tool, but as soon as an update runs, it reverts to the current channel. 
When I use Apps for enterprise in the same scenarios, I can reliably get the installation to remain on semi-annual. It may be by design that Apps for business does not support customising update channels and you are forced onto current, but no documentation that I can find states this conclusively. Please update the docs to make this crystal clear to people using apps for business.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0cd85e3b-e780-457e-9502-856bc6315bca
* Version Independent ID: 416e351f-13c5-e951-ea81-1650910a7c25
* Content: [Overview of update channels for Microsoft 365 Apps - Deploy Office](https://docs.microsoft.com/en-us/deployoffice/overview-update-channels)
* Content Source: [DeployOffice/overview-update-channels.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/overview-update-channels.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by wardkeller** [comment]
@DHB-MSFT Thanks for your quick reply and thank you for confirming that Group Policy and the C2RClient methods are not supported. Our tenant is configured with Semi-Annual as its update channel as you describe, so I am puzzled why when running an update post install using ODT that it changes to current? 
I'll have to test this again. 
Can you shed any light on what tells office to sit on a particular channel? Is there a reg key I can query?

**Comment by manoth-msft** [comment]
Check out this blog post: https://techcommunity.microsoft.com/t5/office-365-blog/how-to-manage-office-365-proplus-channels-for-it-pros/ba-p/795813, search for "Why does this guidance". Scroll y bit down and you will find a table with the locations in which the update channel is configured. It also lists the order of priority, so which location determines the final "winner" of the update channel selection. This should enable you to identify where the actual configuration is coming from.

**Comment by wardkeller** [comment]
Hi @manoth-msft , I appreciate your reply. I had googled and googled and not found that article - thank you! I think this confirms everything. I also noticed that a few days after changing the update channel preferences in the admin centre that my test Windows 10 machine prompted for updates, after updates and a reboot, it was then on semi-annual. So it seems for Apps for business that setting your preference in the admin centre is all that is needed to eventually move installations over to the correct channel. I also tested a machine that was installed with Apps for Enterprise, then signed out of the enterprise licensed user and signed in with an business licensed user. Initially, the update channel was current when it first switched, but then after an update (assuming once the scheduled task ran), it flicked over to semi-annual. This is great because it seemed that I may have had to re-deploy using ODT to get the desired result but all I have to do is change users licenses.
